lms/djangoapps/mobile_api/users/views.py

```diff
@@ -340,6 +341,10 @@ def get_queryset(self):
         ).order_by('created').reverse()
         org = self.request.query_params.get('org', None)
 
+        ## .. filter_implemented_name: CourseEnrollmentQuerysetRequested
+        ## .. filter_type: org.openedx.learning.course_enrollment_queryset.requested.v1
+        enrollments = CourseEnrollmentQuerysetRequested.run_filter(context=enrollments)
```

Comment:
you're missing catching the filter exception. Please, follow [this implementation](https://github.com/openedx/edx-platform/blob/9a0fbe1a5758e80cd378b52bc93ef245ad458069/common/djangoapps/student/models.py#L1682) as a guide

Reply:
Done @mariajgrimaldi